docs/installation/install_epinio_cli.md

```diff
@@ -61,6 +61,60 @@ Move the binary to your PATH
 sudo mv ./epinio /usr/local/bin/epinio
 ```
 
+## Verify Downloaded Files
+
+This is done using the `cosign` tool.
+The following commands were tested using cosign version 2.1.1.
+
+### Verify File Checksum Signature
+
+Instead of signing all release assets, Epinio signs a checksums file containing the different
+release assets checksums. You can download/copy the three files 'epinio_1.8.1_checksums.txt.pem',
+'epinio_1.8.1_checksums.txt.sig', 'epinio_1.8.1_checksums.txt' from the latest release.
+
+```
+curl -LO https://github.com/epinio/epinio/releases/download/v1.8.1/epinio_1.8.1_checksums.txt.pem
+...
```

Comment:
Just as a suggestion, perhaps it may be useful to repeat the curl command with the other options to ease the user copy-paste:
```
curl -LO https://github.com/epinio/epinio/releases/download/v1.8.1/epinio_1.8.1_checksums.txt.pem
curl -LO https://github.com/epinio/epinio/releases/download/v1.8.1/epinio_1.8.1_checksums.txt.sig
curl -LO https://github.com/epinio/epinio/releases/download/v1.8.1/epinio_1.8.1_checksums.txt
```

Reply:
Addressed with commit 48772af701ddd8b727ffa946136ebf5941e9a2c9